community/wine_gecko/APKBUILD

```diff
@@ -3,30 +3,22 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=wine_gecko
 pkgver=2.47
-pkgrel=0
+pkgrel=1
 pkgdesc="Wine's built-in replacement for Microsoft's Internet Explorer"
-url="http://wiki.winehq.org/Gecko"
+url="https://wiki.winehq.org/Gecko"
 arch="x86 x86_64"
 license="MPL"
-options="!archcheck"
+options="!archcheck !check"
```

Comment:
makes sense to mention why no tests possible

Reply:
Done